README.md

```diff
@@ -18,10 +26,13 @@
 *  the tag `v3.x.x` is cut from the `release/v3.0` branch for Rancher v2.7.x line
```

Comment:
While you're here, please insert a "the" before each of the "Rancher v2.X.x line" phrases